docs/tasks/administer-cluster/configure-pod-disruption-budget.md

```diff
@@ -40,13 +42,20 @@ disruption budget to be violated.
 
 ## Specifying a PodDisruptionBudget
 
-A `PodDisruptionBudget` has two components: a label selector `selector` to specify the set of
-pods to which it applies, and `minAvailable` which is a description of the number of pods from that
+A `PodDisruptionBudget` has three components, of which two must be specified: 
+* A label selector `selector` to specify the set of
+pods to which it applies. This is a required field.
+* `minAvailable` which is a description of the number of pods from that
 set that must still be available after the eviction, i.e. even in the absence
 of the evicted pod. `minAvailable` can be either an absolute number or a percentage.
-So for example, 100% means no voluntary evictions from the set are permitted. In
+* If you are using Kubernetes v1.7 or higher, `maxUnavailable` which is a description of the number of pods from that
+set that can be unavailable after the eviction. `maxUnavailable` can also be either an absolute number or a percentage.
+
+You can specify only one of `maxUnavailable` and `minAvailable` in a single Pod Disruption Budget.
+
+So for example, a `minAvailable` of 100% means no voluntary evictions from the set are permitted. Correspondingly, a `maxUnavailable` of 0% means the same. In
```

Comment:
Perhaps some minor changes below. :)

If you are using Kubernetes v1.7 or higher, `maxUnavailable` ~~which~~ is a description of the number of pods from that set that can be unavailable after the eviction. ~~`maxUnavailable` can also~~ It can be either an absolute number or a percentage.